go/test/endtoend/vtgate/queries/misc/misc_test.go

```diff
@@ -54,6 +55,97 @@ func start(t *testing.T) (utils.MySQLCompare, func()) {
 	}
 }
 
+func BenchmarkBlockJoin(b *testing.B) {
+	mcmp, closer := start(b)
+	defer closer()
+
+	type Rep struct {
```

Comment:
add in the comments the output for running the benchmark and also the command used to run it.